src/main/java/mServer/tool/MserverDaten.java

```diff
@@ -31,13 +32,19 @@
     public static MserverListeSuchen listeSuchen = new MserverListeSuchen();
     public static MserverListeUpload listeUpload = new MserverListeUpload();
     public static boolean debug = false;
+    public static boolean restart = false;
```

Comment:
Wieso public static? So kann es jeder einfach so ändern. Lieber sollte man aus der Klasse ein Singelton machen und dann nur via Methoden zugriff gewähren, so würden wir uns an die Kapselung halten und es wird wart- & lesbarer. Zusätzlich geben wir so weniger über die Klasse preis und können eherm al was von der Funktionsweise ändern ohne direkt bei jedem aufrufer es ändern zu müssen. Beispiel wenn wir den Wert in Zukunft aus einer Konfigurationsdatei lesen wollen können wir das einfach napassen und die getter & setter bleiben nach außen hin gleich aber intern lesen und schreiben sie in eine Konfigdatei o.ä.